client/components/fields/description.jsx

```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+
+export function description({item}) {
```

Comment:
@nrvikas es6 react func you mean `export const description = ({item) => ...` ?

Reply:
Yes please.